effect/jvm/src/test/scala/scalaz/effect/RTSTest.scala

```diff
@@ -326,14 +326,14 @@ class RTSSpec(implicit ee: ExecutionEnv) extends Specification with AroundTimeou
     if (n <= 0) 0
     else n + sum(n - 1)
 
-  def deepMapPoint(n: Int): IO[Throwable, Int] =
-    if (n <= 0) IO.point(n) else IO.point(n - 1).map(_ + 1)
+  def deepMapPoint(n: Int, ctr: Int): IO[Throwable, Int] =
```

Comment:
Should this be renamed to `deepBindPoint`?

Reply:
Fair. Will do!